wzdx/sample_files/enhanced/planned_events/prod_snapshot_2022_09_23.json

```diff
@@ -0,0 +1,6407 @@
+{
```

Comment:
Do we need this snapshot for some test?

Reply:
Nope, I removed it, but I left the most recent one (always useful to have an up to date sample)